src/lib/Layout/PanelSideBarLayout/PanelSideBar/Context/PanelSideBarContext.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { ComponentType, createContext, ReactNode, useContext, useState } from "react";
+import { PanelItem } from "../Definitions/PanelItem";
+
+export interface PanelLinkRendererProps<T> {
+  item: PanelItem<T>;
```

Comment:
same with this props, as they are exported.

Reply:
@neotob Exposed props commented

Reply:
awesome thank you!